reference/compound_stmts.po

```diff
@@ -155,13 +224,21 @@ msgid ""
 "continues with the next item, or with the :keyword:`else` clause if there is "
 "no next item."
 msgstr ""
+"Une instruction :keyword:`break` exécutée dans la première suite termine la "
+"boucle sans exécuter la suite de la clause :keyword:`else`. Une instruction :"
+"keyword:`continue` exécutée dans la première suite saute le reste de la "
+"suite et continue avec l'élément suivant, ou avec la clause :keyword:`else` "
+"s'il n'y a pas d'élément suivant."
 
 #: ../Doc/reference/compound_stmts.rst:179
 msgid ""
 "The for-loop makes assignments to the variables(s) in the target list. This "
 "overwrites all previous assignments to those variables including those made "
 "in the suite of the for-loop::"
 msgstr ""
+"La boucle for effectue des affectations aux variables de la liste cible, ce "
```

Comment:
Pourquoi pas mettre for entre double backticks pour indiquer que c'est une instruction Python ?

Reply:
:+1: